src/mjml/mjml.service.js

```diff
@@ -46,7 +46,11 @@ export default class MjmlService {
    * @todo show validation erros in the UI
    * @returns string
    */
-  static mjmlToHtml(mjml) {
+  static mjmlToHtml(mjml, endpoint = null) {
```

Comment:
endpoint is a string. therefore it should be: `endpoint = ''` and the check is endpoint === ''`

Reply:
done